openreview/venue/recruitment.py

```diff
@@ -94,7 +94,17 @@ def invite_committee(self,
             invited_group_ids=list(set(invited_roles) & set(memberships))
             member_group_ids=list(set(member_roles) & set(memberships))
 
-            if invited_group_ids:
+            profile_emails = []
+            profile = None
+            if email.startswith('~'):
```

Comment:
what happens if the profile id doesn't exist?

Reply:
then it won't enter the first if because there is no profile, and the error will be caught later on. I guess it would be better to just catch it here as well?